app/components/works/globus_setup_component.html.erb

```diff
@@ -4,7 +4,7 @@
          <p><strong>You have initiated a Globus-mediated deposit. Here are your next steps. 
             If you need help, visit the <%= link_to 'help page', Settings.globus.help_doc_url, target: '_blank' %>.</strong>
          </p>
-         <ul>
+         <ol>
```

Comment:
This message was updated in a previous issue, but I neglected to make in an ordered list then. So adding that in now.